app/frontend/src/styles/application/form.scss

```diff
@@ -67,6 +67,21 @@
   }
 }
 
+.inline-form-group {
+  display: inline-block;
+
+  &.inline-grade {
```

Comment:
i would either name these classes more generically or move them to a scss file specific to what they are specific too